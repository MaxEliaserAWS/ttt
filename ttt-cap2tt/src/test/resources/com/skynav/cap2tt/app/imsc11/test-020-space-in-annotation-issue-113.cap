Lambda字幕V4	DF0+1	SCENE"和文標準"

1485	01074610/01075219	終わりが来たから	＠縦右 ＠行頭	＠斜３
				＿＿＿_僕は行くけど
1402	01481418/01481823	＿＠斜３［みんな 歌って］＠
				＠斜３［“すばらしい この世界”］＠
1485	01574610/01575219	終わりが来たから	＠縦右 ＠行頭	＠斜３
				＿＿＿_僕は行くけど
