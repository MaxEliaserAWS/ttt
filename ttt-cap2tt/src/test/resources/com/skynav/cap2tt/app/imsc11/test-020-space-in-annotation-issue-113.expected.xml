<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:shear="16.78842%"/><style xml:id="s2" tts:textAlign="center"/><style xml:id="s3" tts:textAlign="start"/><style xml:id="s4" tts:fontStyle="oblique"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/><region xml:id="縦右" tts:extent="15rw 80rh" tts:position="right 10rw center" tts:writingMode="tbrl"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="01:07:46:10" end="01:07:52:19" region="縦右" style="s1">終わりが来たから<br/>　　　 僕は行くけど</p><p begin="01:48:14:18" end="01:48:18:23" style="s2"><span style="s3">　<span style="s4">みんな 歌って</span><br/><span style="s4">“すばらしい この世界”</span></span></p><p begin="01:57:46:10" end="01:57:52:19" region="縦右" style="s1">終わりが来たから<br/>　　　 僕は行くけど</p></div></body></tt>